### PR TITLE
Fix: Update prometheus_server readme.rst

### PR DESCRIPTION
**Pull request by sobrase**
Added yml file for prometheus_server role used to configure prometheus

### CODE REVIEW

**Comment by oxedions** [comment]
Can you add a yaml code block for this ?

Also please remove the  :: and only add a single :

File: roles/addons/prometheus_server/readme.rst [hunk 1]
```diff
@@ -15,6 +15,21 @@ Alertmanager is available at http://localhost:9093
 
 Node_exporter is available at http://localhost:9100
 
+In order to allow the role to deploy, you need to add the following 
+content under a file inventory/group_vars/all/addons/monitoring.yml :
```
Comment (btravouillon) [inline roles/addons/prometheus_server/readme.rst:19]: I would rather prefer to set defaults/main.yml, which would allow the role to deploy out-of-the-box.

**Comment by osmocl** [comment]
Hello
Sorry to bother you :)
I was wondering if you could provide some exporters examples in the readme too.

```
# cat monitoring.yml
monitoring:

  exporters:
    node_exporter:
      package: node_exporter
      service: node_exporter
[...]

File: roles/addons/prometheus_server/readme.rst (hunk 1 above)
Comment (oxedions) [reply roles/addons/prometheus_server/readme.rst:19]: Deal !

**Comment by oxedions** [comment]
Done in 330.

**Pull request closed without merging**